workers/deployment_manager/SnapshotGenerator/Program.cs

```diff
@@ -0,0 +1,46 @@
+using System;
```

Comment:
Do you need the SnapshotGenerator?

Reply:
just to generate the initial empty snapshot for the deployment manager deployment

Reply:
guess I can just remove it for now ;)